docker/README.md

```diff
@@ -6,117 +6,157 @@
 
 - [Athenz on Docker](#athenz-on-docker)
     - [Index](#index)
-    - [Prerequisite](#prerequisite)
+    - [Prerequisites](#prerequisites)
     - [Build Athenz](#build-athenz)
     - [Deploy Athenz](#deploy-athenz)
+    - [Verify Athenz Deployment](#verify-athenz-deployment)
+        - [JAVA Remote debugging](#java-remote-debugging)
     - [Cleanup](#cleanup)
-    - [Configuration Details](#configuration-details)
-    - [Useful Commands](#useful-commands)
-    - [[WIP] deploy with docker-stack](#wip-deploy-with-docker-stack)
-    - [TO-DO](#to-do)
-    - [Important Files](#important-files)
+    - [Appendix](#appendix)
+        - [Important Files](#important-files)
+        - [[WIP] Configuration Details](#wip-configuration-details)
+        - [Useful Commands](#useful-commands)
+        - [TODO](#todo)
 
 <!-- /TOC -->
 
-<a id="markdown-prerequisite" name="prerequisite"></a>
-## Prerequisite
-- docker
-- make
+<a id="markdown-prerequisites" name="prerequisites"></a>
+## Prerequisites
+
+1. `git`
+1. `docker`
+1. `make`
 
 <a id="markdown-build-athenz" name="build-athenz"></a>
 ## Build Athenz
 
 ```bash
 cd `git rev-parse --show-toplevel`/docker
 
-# there are a lot of build logs. You may want to check it inside a log file later on.
-make build | tee ./athenz-docker-build.log
+# it takes about 15-30 mins
+make build
+
+# TODO: update the min. version before merge
+# P.S. the latest code may cause docker build to fail, please use older version by specifying the tag version (> v1.8.60) or post an issue
+# make build TAG=v1.8.60
```

Comment:
Plx help to update the README.md, when the tag version of merging this PR is known.